.github/workflows/github.yml

```diff
@@ -13,6 +13,7 @@ jobs:
   build:
     name: Build (${{ matrix.platform }})
     strategy:
+      fail-fast: false
```

Comment:
Leftovers from testing?

Reply:
I have added it to always compile other builds, currently it's quick, so doesn't matter if one build fails. Others can be completed.